node/src/blockchain/blockchain_interface.rs

```diff
@@ -926,15 +1033,23 @@ mod tests {
             TEST_DEFAULT_CHAIN,
         );
 
+        let start_block = BlockNumber::Number(42u64.into());
         let result = subject.retrieve_transactions(
-            42,
+            start_block,
+            BlockNumber::Latest,
             &Wallet::from_str("0x3f69f9efd4f2592fd70be8c32ecd9dce71c472fc").unwrap(),
         );
 
+        let new_start_block = if let BlockNumber::Number(start_block_nbr) = start_block {
```

Comment:
It would have helped me understand this code if this variable had been called something like `expected_fallback_start_block`.

Reply:
ok, I used that name to take advantage of `new_start_block: new_start_block` initialization shorthand. `expected_fallback_start_block` probably makes the test more readable to see where it's used.